synctl

```diff
@@ -3715,13 +3958,56 @@ def main():
+            if get_args.active is not None:
+                new_payload = update_instance.update_active(payload[0], get_args.active)
+            if get_args.timeout is not None:
+                new_payload = update_instance.update_config_timeout(payload[0], get_args.timeout)
+            if get_args.retries is not None:
+                new_payload = update_instance.update_retries(payload[0], get_args.retries)
+            if get_args.frequency is not None:
+                new_payload = update_instance.update_frequency(payload[0], get_args.frequency)
+            if get_args.retry_interval is not None:
+                new_payload = update_instance.update_retry_interval(payload[0], get_args.retry_interval)
+            if get_args.operation is not None:
+                new_payload = update_instance.update_ping_operation(payload[0], get_args.operation)
+            if get_args.script_file is not None:
+                new_payload = update_instance.update_config_script_file(payload[0], get_args.script_file)
+            if get_args.description is not None:
+                new_payload = update_instance.update_description(payload[0], get_args.description)
+            if get_args.record_video is not None:
+                new_payload = update_instance.update_record_video(payload[0], get_args.record_video)
+            if get_args.location is not None:
+                new_payload = update_instance.update_locations(payload[0], get_args.location)
+            if get_args.mark_synthetic_call:
+                new_payload = update_instance.update_mark_synthetic_call(payload[0],
+                                                                         get_args.mark_synthetic_call)
+            if get_args.entry_file is not None:
+                new_payload = update_instance.update_script_file(payload[0], get_args.entry_file)
+            if get_args.url is not None:
+                new_payload = update_instance.update_url(payload[0], get_args.url)
+            if get_args.follow_redirect is not None:
+                new_payload = update_instance.update_follow_redirect(payload[0], get_args.follow_redirect)
+            if get_args.expect_status is not None:
+                new_payload = update_instance.update_expect_status(payload[0], get_args.expect_status)
+            if get_args.validation_string is not None:
+                new_payload = update_instance.update_validation_string(payload[0], get_args.validation_string)
+            if get_args.bundle is not None:
+                new_payload = update_instance.update_bundle(payload[0], get_args.bundle)
+            if get_args.custom_property is not None:
+                split_string = get_args.custom_property.split(',')
+                new_payload = update_instance.update_custom_properties(payload[0], split_string)
+            update_instance.update_a_synthetic_test(get_args.id, json.dumps(new_payload))
+        if get_args.syn_type == SYN_ALERT:
```

Comment:
Those options supported by `update test` is not supported by `update alert`, we can show a waning message when user specify any of those options. like:
`synctl update alert `id` --active false`, 

`--active` is not support by `update alert`